brave/browser-laptop
# Moves bookmark text calculation into the state

Submitter Checklist:


Resolves #9517

Auditors:

Test Plan:

0) add so many bookmarks that you see bookmark overflow icon 
1) check if overflow is displayed correctly
2) open two windows and try to resize them, check if overflow is displayed correctly
3) try reorder bookmarks from overflow to toolbar and see if overflow is still displayed correctly


Reviewer Checklist:

Tests
